Change yahboom documentation to modular. (#1921)
viamrobotics/docs: docs/components/arm/yahboom-dofbot.md

linkTitle: "yahboom-dofbot"
weight: 50
type: "docs"

> draft: "true"

Reviewer: does it need to be a draft?

Author: Until I publish the module - I'd rather yes - also I wasn't sure if you were documenting viam-labs modules, which are meant to be less well-vetted than the ones in the viam namespace.

> description: "Configure a yahboom dofbot modular arm."

Reviewer 2: ```suggestion
description: "Configure a Yahboom DOFBOT modular arm."
```
If it's not the model name anymore -> sentence case

images: ["/icons/components/arm.svg"]
tags: ["arm", "components"]
---

> Viam provides a `dofbot` arm from the manufacturer `yahboom` as a modular resource. To add a [Yahboom DOFBOT](https://category.yahboom.net/collections/r-robotics-arm) to your robot:

Reviewer 2: ```suggestion
Viam supports the [Yahboom DOFBOT](https://category.yahboom.net/collections/r-robotics-arm) arm as a modular resource.
Configure a `dofbot` arm to add it to your robot:
```

Reviewer: ```suggestion
Viam supports the [Yahboom DOFBOT](https://category.yahboom.net/collections/r-robotics-arm) arm as a [modular resource](https://github.com/viam-labs/yahboom).
Configure a `dofbot` arm to add it to your robot:
```

{{< tabs >}}
{{% tab name="Config Builder" %}}

Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).

> Click on the **Components** subtab and find the **Create component** pane.

Reviewer 2: ```suggestion
Click on the **Components** subtab and click **Create component**.
```
Per recent UI change

Select the `arm` type, then select the `rand:yahboom:dofbot` modular resource.
Enter a name for your arm and click **Create**.

> Note: There are not attributes for this modular arm.

Reviewer 2: ```suggestion

There are no attributes available for this modular arm.
```

{
  "components": [
    {

>       "model": "dofbot",
>       "name": "<your-arm-name>",

Reviewer 2: ```suggestion
      "name": "<your-arm-name>",
      "model": "dofbot",
```
[nit] For consistency with PR 1920
